src/ui/ResponsiveComponents/ResponsiveFacets.ts

```diff
@@ -36,9 +37,9 @@ export class ResponsiveFacets implements IResponsiveComponent {
     ResponsiveComponentsManager.register(ResponsiveFacets, $$(root), Facet.ID, component, options);
   }
 
-  constructor(public coveoRoot: Dom, public ID: string, options: IResponsiveComponentOptions) {
+  constructor(public coveoRoot: Dom, public ID: string, options: IResponsiveComponentOptions, dropdownMock?: ResponsiveDropdown) {
```

Comment:
I would just call the parameter `responsiveDropdown`, without the `mock`. It's just a mock for test, not all the time.